dts/arm/nrf51_vbluno51.dts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2017 VNG IoT Lab Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf51822.dtsi>
+
+/ {
+	model = "VNG VBLUno51 BLE board";
+	compatible = "nordic,pca10028-dk", "nordic,nrf51822-qfac",
```

Comment:
This should be:

compatible = "vng,vbluno51", "nordic,nrf51822-qfac", "nordic,nrf51822";

(most specific to least)